desktop-apps.md

```diff
@@ -179,41 +179,43 @@ https://github.com/LostRuins/koboldcpp is a fun twist on LLMs -- adding game lik
 The UI is pretty basic -- and you get some surprising answers. Here I ask a simple icebreaker question -- and you see that it responds that it is a friendly AI that likes to play games.
 
 ```{figure} https://static.premai.io/book/desktopapps-koboldcpp-ui.png
-Koboldcpp UI
+koboldcpp UI
 ```
 
 ### Scenarios
 
 You can also enter different sorts of scenarios and modes.
 
 ```{figure} https://static.premai.io/book/desktopapps-kcpp-scenarios.png
-Koboldcpp Scenarios
+koboldcpp Scenarios
 ```
 
 Below is the Julius Caesar scenario!
 
 ```{figure} https://static.premai.io/book/desktopapps-kcpp-jc.png
-Koboldcpp Julius Caesar Chat
+koboldcpp Julius Caesar Chat
 ```
 
 ### Model Configuration and Tools
 
 Many of the model configurations are similar to the default that is offered. But there are some interesting twists like story mode, adventure mode, and instruct mode.
 
 ```{figure} https://static.premai.io/book/desktopapps-kcpp-modes.png
-Koboldcpp Julius Model Configuration
+koboldcpp Julius Model Configuration
 ```
 
-## [Local.AI](https://local.ai)
+## [local.ai]
 
-[Local.AI](https://local.ai) is a simple app for loading LLMs after you manually download a `ggml` model from online.
+[local.ai]: https://www.localai.app
+
+The [local.ai] App from https://github.com/louisgv/local.ai ([not to be confused](https://github.com/louisgv/local.ai/discussions/71) with [LocalAI](https://localai.io) from https://github.com/go-skynet/LocalAI) is a simple application for loading LLMs after you manually download a `ggml` model from online.
 
 ### UI and Chat
 
 The UI and chat are pretty basic. One bug that I noticed was that it wasn't possible to load models from the UI -- I had to manually download the model and then use the app.
 
 ```{figure} https://static.premai.io/book/desktopapps-localai-ui.png
-[Local.AI](https://local.ai) UI
+[local.ai] UI
```

Comment:
Let's add the correct link [here](https://www.localai.app/)

Reply:
already added in line 209? (using markdown syntax it's reused here).